paws.common/DESCRIPTION

```diff
@@ -23,6 +25,7 @@ Imports:
     jsonlite,
     methods,
     utils,
+    Rcpp,
```

Comment:
I think my biggest question is does Rcpp need to be an import, or can it be a build time dependency (whatever that means in R)? I think for Windows users in particular, Rcpp is a lot of extra stuff to download/install.

For example dplyr which definitely uses C++ does not have Rcpp as an import and I would like to figure out how they do it.

Reply:
That is a good question. I will have a look.

Reply:
According to https://cran.r-project.org/web/packages/Rcpp/vignettes/Rcpp-package.pdf, Rcpp is required both in Imports and LinkingTo. 

dplyr doesn't have it because it got rid of Rcpp in version 1.0.0 and replaced it with `vctrs` which uses C and I guess R standard library instead of C++ and Rcpp.

I think we'll need to figure out the impact on build time and on Windows users though.

Reply:
I think this stackoverflow answers some of our concerns. https://stackoverflow.com/questions/41434895/does-installing-an-r-package-with-rcpp-require-rtools-windows

Once cran has created the binaries windows computers can install the package without the need for Rtools. However if a user tries to install the package from source then Rtools are required.

From my understanding there is alot of packages that utilises rcpp nowadays. There is another c++ interface for R cpp11 which offers similar support to Rcpp. However I believe rtools will be required when installing from source.

Reply:
as an FYI - dplyr _used_ to depend on Rcpp and BH but before 1.0 made a huge push to remove all need for that with their own implementation + stuff out of vctrs package to improve compilation performance and more tightly control the dep tree